nextcord/types/user.py

```diff
@@ -10,6 +10,7 @@ class PartialUser(TypedDict):
     username: str
     discriminator: str
     avatar: Optional[str]
+    global_name: str
```

Comment:
According to the [docs](https://discord.com/developers/docs/resources/user#user-object-user-structure), this field can be null.

Reply:
I had left that out because it seemed like the display name is just always set, probably defaults to the username or something?

Reply:
Scratch that, looks like _my_ display name was set to the same thing as my username for some reason 😅